src/main/java/se/sundsvall/oepintegrator/integration/opene/rest/OpeneRestClient.java

```diff
@@ -36,4 +37,8 @@ Optional<byte[]> getCaseListByFamilyId(
 		@PathVariable(name = "status", required = false) final String status,
 		@RequestParam(name = "fromDate", required = false) final String fromDate,
 		@RequestParam(name = "toDate", required = false) final String toDate);
+
+	@GetMapping(path = "/api/instanceapi/getinstance/{flowInstanceId}/pdf")
+	Response getCasePdfByFlowInstanceId(@PathVariable("flowInstanceId") String flowInstanceId);
```

Comment:
Is the expected response from the integration a binary file?

If so: `ResponseEntity<InputStreamResource>`